assets/tests/components/map.test.tsx

```diff
@@ -330,9 +331,25 @@ describe("<Map />", () => {
       screen.queryByRole("switch", { name: /Street View/ })
     ).not.toBeInTheDocument()
   })
+
+  test("sets selected vehicle id as selected ", () => {
+    const vehicle = vehicleFactory.build()
+
+    const { container } = render(
+      <Map vehicles={[vehicle]} selectedVehicleId={vehicle.id} />
+    )
+
+    expect(
+      container.querySelector(".m-vehicle-map__icon .selected")
+    ).toBeInTheDocument()
+    expect(
+      container.querySelector(".m-vehicle-map__label.selected")
+    ).toBeInTheDocument()
```

Comment:
If you have time and feel like it, I'd be happy to look at a PR for that refactor! Otherwise I agree that these assertions aren't _great_ but I don't think they're dealbreakers.